hypervisor/lib/string.c

```diff
@@ -22,7 +22,7 @@
 #define LONG_MIN        ((long)(~LONG_MAX))             /* 0x80000000 */
 #endif
 
-#define ISSPACE(c) (((c) & 0xFFU == ' ') || ((c) & 0xFFU == '\t'))
+#define ISSPACE(c) ((((c) & 0xFFU) == ' ') || (((c) & 0xFFU) == '\t'))
```

Comment:
Please remove following too:
#ifndef ULONG_MAX

#ifndef LONG_MAX


There are no other defines now.

Reply:
@dongyaozu have updated, please review, thanks for comments.
-Yonghua